packages/core/src/PlayerContextProvider.js

```diff
@@ -991,6 +993,7 @@ export class PlayerContextProvider extends Component {
 PlayerContextProvider.propTypes = {
   playlist: PropTypes.arrayOf(PlayerPropTypes.track.isRequired).isRequired,
   autoplay: PropTypes.bool.isRequired,
+  createMediaElement: PropTypes.func,
```

Comment:
You can go ahead and mark this `isRequired` since it shouldn't ever not have a value.